src/dataExplorer/components/SaveAsCellForm.tsx

```diff
@@ -170,21 +176,37 @@ class SaveAsCellForm extends PureComponent<Props, State> {
 
     const viewWithProps: View = {...view, name: cellName}
 
+    const redirectHandler = (dashboardId: string): void =>
+      history.push(`/orgs/${orgID}/dashboards/${dashboardId}`)
+
     try {
-      targetDashboardIDs.forEach(dashID => {
+      targetDashboardIDs.forEach((dashID, i) => {
+        const toRedirect =
+          i === targetDashboardIDs.length - 1 ? redirectHandler : undefined
```

Comment:
you typically don't want to set values to `undefined`, you want to use `null`. `undefined` means "hasn't been assigned a value." `null` means no value. you don't assign a variable an `undefined` value, you assign it `null`. 

```ts
let foo;
foo; // undefined
```